samples/camera-capture/app.js

```diff
@@ -23,10 +23,46 @@ Author: Eric Bidelman (ericbidelman@chromium.org)
  *
  * @see http://developer.chrome.com/apps/manifest.html#permissions
  */
+
+var curStream = null; // keep track of current stream
+
 function getCamera() {
-  navigator.webkitGetUserMedia({audio: true, video: true}, function(stream) {
-    document.querySelector('video').src = URL.createObjectURL(stream);
+  var cameraSrcId = document.querySelector('select').value;
+
+  // constraints allow us to select a specific video source 
+  var constraints = {
+    video: {
+      optional: [{
+        sourceId: cameraSrcId
+      }]
+    },
+    audio:true
+  }
+
+  // stop any previous stream (eg. while switching between cameras)
+  if(curStream) {
+    curStream.stop();
+  }
+
+  navigator.webkitGetUserMedia(constraints, function(stream) {
```

Comment:
It would be nice to add something like: `
navigator.getUserMedia = navigator.getUserMedia || navigator.webkitGetUserMedia;` before.